homeassistant/components/powerwall/__init__.py

```diff
@@ -239,6 +241,10 @@ async def _call_base_info(power_wall: Powerwall, host: str) -> PowerwallBaseInfo
     except BaseExceptionGroup as e:
         raise e.exceptions[0] from None
 
+    battery_map: dict[str, BatteryResponse] = {
```

Comment:
I think you can drop the typing as mypy should be able to work it out without having to hint it

Reply:
It's simple enough that you could inline it below since it's only used once